scripts/build_graph_from_csv.py

```diff
@@ -0,0 +1,72 @@
+from matplotlib import pyplot as plt
```

Comment:
Would be great to add `#! /usr/bin/env python3` as a first line, make file executable and remove `.py` extensions to make it more like other scripts